test/library/gui/widgets.EditField.lua

```diff
@@ -29,36 +32,43 @@ function test.editfield_cursor()
     e:onInput{CUSTOM_HOME=true}
     expect.eq(1, e.cursor, 'cursor should be at beginning of string')
     e:onInput{CUSTOM_CTRL_RIGHT=true}
-    expect.eq(6, e.cursor, 'goto beginning of next word')
+    expect.eq(5, e.cursor, 'goto end of current word')
```

Comment:
I think I was mimicking readline behavior before, but I agree with this change since it matches browser edit field behavior.